### PR TITLE
Fix 63/64 EIP-150 problem

### PR DESCRIPTION
**Pull request by Uxio0**

### CODE REVIEW

File: gnosis/safe/safe.py [hunk 1]
```diff
@@ -431,32 +431,40 @@ def estimate_tx_gas_with_web3(self, to: str, value: int, data: bytes) -> int:
         """
         Estimate tx gas using web3
         """
-        return self.ethereum_client.estimate_gas(self.address, to, value, data)
+        gas_estimated = self.ethereum_client.estimate_gas(self.address, to, value, data)
+        for _ in range(20):  # Test `call()` and increase gas 20 times, 63/64th problem
+            try:
```
Comment (rmeissner) [inline gnosis/safe/safe.py]: This will not work for delegate calls right?

Comment (Uxio0) [reply gnosis/safe/safe.py]: No, delegate calls are disabled for his kind of estimation: https://github.com/gnosis/gnosis-py/blob/eip-150-fix/gnosis/safe/safe.py#L460

Comment (rmeissner) [reply gnosis/safe/safe.py]: it this does not throw then we just perform the same call 20 times?

Comment (Uxio0) [reply gnosis/safe/safe.py]: @rmeissner You are right, I fixed it

File: gnosis/safe/safe.py [hunk 2]
```diff
@@ -431,32 +431,41 @@ def estimate_tx_gas_with_web3(self, to: str, value: int, data: bytes) -> int:
...
+                self.w3.eth.call({'gas': gas_estimated, 'from': self.address,
+                                  'to': to, 'value': value,
+                                  'data': data})
+                return gas_estimated
```
Comment (Uxio0) [inline gnosis/safe/safe.py]: @rmeissner It should be alright now

**Pull request merged**